e2e/tests/home-page.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { expect, test } from '@playwright/test';
+import { HOMEPAGE_ENDPOINT } from '../constants';
+
+test.describe('Homepage', () => {
+  test('Should load the homepage', async ({ page }) => {
+    await page.goto(HOMEPAGE_ENDPOINT);
+    await expect(page.getByRole('link', { name: 'Give a gift' })).toBeVisible();
+    await expect(page.getByRole('link', { name: 'Free returns' })).toBeVisible();
+    await expect(page.getByRole('link', { name: 'Sign up for our newsletter' })).toBeVisible();
```

Comment:
None of these are actually in the codebase — they are loaded by the `getNavigation` query. We only want to test things that are in the penny code itself, so you might go to the homepage and then check that some wrapper has loaded. Or, you could load the `siteCopyrightText` variable from the `config` folder and check if that is visible.

If we did want to test nav items they should be something that can be set like the product name.